src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -99,6 +100,11 @@ internal sealed class SourcePropertySymbol : PropertySymbol, IAttributeTargetSym
                         _sourceName = propertyData.IndexerName;
                     }
                 }
+
+                if (this.ReturnsByRefReadonly)
```

Comment:
Moved a little up as we are inspecting syntax up in the same constructor.

---
In reply to: [113565177](https://github.com/dotnet/roslyn/pull/18715#discussion_r113565177) [](ancestors = 113565177)